wolfcrypt/src/evp.c

```diff
@@ -272,9 +272,6 @@ static int evpCipherBlock(WOLFSSL_EVP_CIPHER_CTX *ctx,
                     ret = wc_AesGcmEncrypt(&ctx->cipher.aes, NULL, NULL, 0,
                               ctx->iv, ctx->ivSz, ctx->authTag, ctx->authTagSz,
                               in, inl);
-                    /* Reset partial authTag error for AAD*/
```

Comment:
This check is done for encrypt and decrypt.
Why is this being removed for encrypt?

Reply:
`AES_GCM_AUTH_E` in the encrypt case is only returned when using hardware acceleration. In the normal case, ignoring this error doesn't make sense for encryption since there is no `authTag` to compare to. Should I leave it in for the hardware cases?

Reply:
Leave it in as there is a case when it needs to be there!